src/PageContent/Settings/OrganizationDetails.js

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from 'react';
+import Map from '../../Map/Map.js';
+
+class OrganizationDetails extends Component {
+
+    render() {
+        return (
+            <div className="scs-0">
+                <div className="scs-0-image">
+                    <Map
+                        marginLeft="20px" 
+                        height="150px" 
+                        width="70%" 
+                        marginTop="10px" 
+                        marginRight="0px"
+                        marginBottom="0px"
+                        address={this.props.account.address}
+                    />
+                </div>
+
+                {!this.props.isEditingOrg ?
+
+                    <div className="scs-0-content">
+                        <h5>Organization Details</h5>
+                        <h6>{this.props.account.name}</h6>
+                        <div className="scs-spacing" />
+                        <h6>Address: {this.props.account.address.street1} {this.props.account.address.street2} {this.props.account.address.city}, {this.props.account.address.state} {this.props.account.address.zipcode}</h6>
+                        <h6>Office Number: {this.props.account.organizationPhone}</h6>
+                        {this.props.account.numVolunteers ? 
+                            <h6>Volunteers: {this.props.account.numVolunteers}</h6>
+                            :
+                            <span />
+                        }
+                        {this.props.account.notes ? 
+                            <h6>Notes: {this.props.account.notes}</h6>
+                            :
+                            <span />
+                        }
+                        {this.props.account.emergencypickup ? 
```

Comment:
instead of checking for `acceptEmergencyPickups`, because it can be `true` or `false`, check `accountType == AccountType.RECEIVING_AGENCY`